.happy/terraform/modules/sfn_config/covidhub-import.wdl

```diff
@@ -0,0 +1,32 @@
+version 1.1
+
+workflow covidhub_import {
+    input {
+        String docker_image_id = "covidhub-import"
+        String aws_region = "us-west-2"
+    }
+
+    call covidhub_import_workflow {
+        input:
+        docker_image_id = docker_image_id,
+        aws_region = aws_region,
+    }
+}
+
+task covidhub_import_workflow {
+    input {
+        String docker_image_id
+        String aws_region
+    }
+
+    command <<<
+    echo "RUNNING MIGRATE SCRIPT"
+    chmod +x /usr/src/app/scripts/migrate-all.sh
```

Comment:
nit -- afaik we can just set the mode of this file in the git repo and we won't have to chmod it here.